### PR TITLE
Merge current development to prod

### PR DESCRIPTION
**Pull request by AlwinEgger**
This includes:

- airbrake-js needs to be fetched via npm since maintainer deleted all tags on github :(
- using last available angularjs 1.7.8
- adapted a few versions of deps (vul. correction)
- corrected test (included quill)
- corrected travis node version

- [ ] Version needs to be corrected after merge

### CODE REVIEW

File: app/scripts/app.js [hunk 1]
```diff
@@ -589,38 +589,38 @@ angular
...
+          airbrake.notify(message);
+          return $q.reject(rejection);
         }
-        airbrake.notify(message);
-        return $q.reject(rejection);
-      }
-    };
-  })
+      };
+    })
```
Comment (fabianpage) [inline app/scripts/app.js]: gabs hier nur einen whitespace change?

File: bower.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+    "angular": "~1.7.8",
+    "angular-animate": "~1.7.8",
     "angular-base64": "~2.0.5",
     "angular-bootstrap": "~2.5.0",
     "angular-bootstrap-switch": "~0.5.2",
     "angular-cookie": "~4.1.0",
-    "angular-cookies": "~1.6.10",
+    "angular-cookies": "~1.7.8",
     "angular-gettext": "~2.3.10",
     "angular-i18n": "^1.6.6",
```
Comment (fabianpage) [inline bower.json]: gibt's da keine 1.7.8 version?

Comment (AlwinEgger) [reply bower.json]: Doch

**Review by fabianpage — APPROVED** [review]

File: app/scripts/app.js (hunk 1 above)
Comment (AlwinEgger) [reply app/scripts/app.js]: offenbar

**Review by fabianpage — APPROVED** [review]

**Pull request merged**